src/main/kotlin/com/exactpro/th2/check1/rule/RuleFactory.kt

```diff
@@ -138,6 +138,7 @@ class RuleFactory(
     }
 
     private fun MessageFilter.toRootMessageFilter(): RootMessageFilter {
+        check(this.messageType.isNotBlank()) { "Rule cannot be executed because the message filter does not contain 'message type'" }
```

Comment:
Shouldn't we check the message type for `RootMessageFilter` as well? The CheckRuleRequest/CheckSequenceRuleRequest might contain the RootMessageFilter instead of MessageFilter